rcl/test/rcl/test_events.cpp

```diff
@@ -726,6 +725,10 @@ TEST_P(TestEventFixture, test_sub_message_lost_event)
     return;
   }
 
+  // (TODO blast545): Enable the rest of the test, timeout failing
```

Comment:
Is this going to be addressed in a followup PR? Otherwise I would also create an issue and add the link here.

Also instead of merging in dead code, I would remove this code below and create a separate branch for it.

Reply:
I'd also say that if the issue only occurs when running against `rmw_cyclonedds_cpp`, we can skip the test for that particular implementation.

Reply:
Opened this PR to track this failure: https://github.com/ros2/rcl/pull/705